gtk3/test/test-gtk-tree-iter.rb

```diff
@@ -102,6 +102,15 @@ def test_nth_children
     assert_equal(second_child_iter, @iter.nth_child(1))
   end
 
+  def test_firs_child
+    @iter.values = ["Dad"]
+    first_child_iter = @model.append(@iter)
+    first_child_iter.values = ["First son"]
+    second_child_iter = @model.append(@iter)
+    second_child_iter.values = ["First daughter"]
+    assert_equal(@iter.nth_child(0), @iter.first_child)
```

Comment:
`first_child_iter` is better rather than `@iter.nth_child(0)` because we know `first_child_iter` is the first child in this context.